src/components/share/modal/ShareModal.tsx

```diff
@@ -0,0 +1,378 @@
+import { DropdownMenuOption } from ":/components/dropdown-menu";
+import { InvitationUserSelectorList } from "../InvitationUserSelectorList";
+import {
```

Comment:
To be discussed by Robin I guess, I feel this search part has some UX improvements ahead:

- It does not tell the user what he should do: "Please start typing"
- Showing "Resultat de la recherche" when there is no search started yet feels weird imo - maybe an illustration instead or something saying "the results will be displayed here" in some sort 

Open subject

<img width="915" alt="Capture d’écran 2025-04-16 à 15 15 20" src="https://github.com/user-attachments/assets/c550af66-61fd-447e-bc99-648a115cac4b" />